internal/pkg/cli/pipeline_init.go

```diff
@@ -557,7 +579,7 @@ func buildPipelineInitCmd() *cobra.Command {
 		}),
 	}
 	cmd.Flags().StringVarP(&vars.appName, appFlag, appFlagShort, tryReadingAppName(), appFlagDescription)
-	cmd.Flags().StringVarP(&vars.githubURL, githubURLFlag, githubURLFlagShort, "", githubURLFlagDescription)
+	cmd.Flags().StringVarP(&vars.repoURL, githubURLFlag, githubURLFlagShort, "", githubURLFlagDescription)
```

Comment:
Should we update the rest of these variables that include `github`, or is that out of scope for this PR?

Reply:
Yup, those changes are all in the next PR. :)